_test/test_combine/test_combine_real.m

```diff
@@ -1,146 +1,149 @@
-            mf_qq = mf_qq.set_fun (@fake_cross_sec, 0.9*[this.stiffness,this.gam,this.amp]);
-            [wfit_qq,fitdata_qq] = mf_qq.fit();
-            
-            mf_combi = multifit_sqw (w2d_qq_combined);
-            mf_combi = mf_combi.set_fun (@fake_cross_sec, 0.9*[this.stiffness,this.gam,this.amp]);
-            [wfit_combi,fitdata_combi] = mf_combi.fit();
-
-            [ok,mess]=equal_to_tol(fitdata_qq.p,fitdata_combi.p,-1e-6,'ignore_str', 1);
-            assertTrue(ok,['combine sqw fails: ',mess])
-        end
-        
-        % ------------------------------------------------------------------------------------------------
-        function this = test_combine_dnd_notol(this)
-            % dnd combination without specifying a tolerance
-            w2d_qq_d2d=read_dnd(fullfile(this.testdir,'w2d_qq_d2d.sqw'));
-            w2d_qq_d2d_plus=read_dnd(fullfile(this.testdir,'w2d_qq_d2d_plus.sqw'));
-            w2d_qq_d2d_minus=read_dnd(fullfile(this.testdir,'w2d_qq_d2d_minus.sqw'));
-            
-            w2d_qq_combined=combine_horace_2d(w2d_qq_d2d_minus,w2d_qq_d2d_plus);
-            
-            mf_qq = multifit_sqw (w2d_qq_d2d);
-            mf_qq = mf_qq.set_fun (@fake_cross_sec, 0.9*[this.stiffness,this.gam,this.amp]);
-            [wfit_qq,fitdata_qq] = mf_qq.fit();
-            
-            mf_combi = multifit_sqw (w2d_qq_combined);
-            mf_combi = mf_combi.set_fun (@fake_cross_sec, 0.9*[this.stiffness,this.gam,this.amp]);
-            [wfit_combi,fitdata_combi] = mf_combi.fit();
-            
-            [ok,mess]=equal_to_tol(fitdata_qq.p,fitdata_combi.p,-2.2e-2,'ignore_str', 1);
-            assertTrue(ok,['combine dnd without a specified tolerance fails: ',mess])
-            
-        end
-        
-        % ------------------------------------------------------------------------------------------------
-        function this = test_combine_dnd_tol(this)
-            % dnd combination specifying a tolerance
-            w1d_d1d=read_dnd(fullfile(this.testdir,'w1d_d1d.sqw'));
-            w1d_d1d_plus=read_dnd(fullfile(this.testdir,'w1d_d1d_plus.sqw'));
-            w1d_d1d_minus=read_dnd(fullfile(this.testdir,'w1d_d1d_minus.sqw'));
-            
-            w1d_combined=combine_horace_1d(w1d_d1d_minus,w1d_d1d_plus,0.025);
-
-            mf = multifit_sqw (w1d_d1d);
-            mf = mf.set_fun (@fake_cross_sec, 0.9*[this.stiffness,this.gam,this.amp]);
-            [wfit,fitdata] = mf.fit();
-            
-            mf_combi = multifit_sqw (w1d_combined);
-            mf_combi = mf_combi.set_fun (@fake_cross_sec, 0.9*[this.stiffness,this.gam,this.amp]);
-            [wfit_combi,fitdata_combi] = mf_combi.fit();
-            
-            [ok,mess]=equal_to_tol(fitdata.p,fitdata_combi.p,-2e-2,'ignore_str', 1);
-            assertTrue(ok,['combine dnd with a specified tolerance fails: ',mess])
-            
-        end        
-    end
-end
+classdef test_combine_real < TestCase
```

Comment:
What makes it "real"?

Reply:
test_combine coincides with the name of the folder so causes problems in some cases. I have renamed it on the first word got in mind -- real, because its real data from some published experiment

Reply:
There shouldn't be any problems due to file name/folder collisions, but regardless, I would probably say "experimental", "exp" or similar rather than "real" which, to me, implies there's imaginary data somewhere.